packages/headless-react/typedoc/documents/hooks.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Controller hooks
+---
+
+# Hooks
+
+This package provides React hooks for controllers defined in the Engine Definition. Hooks allow you to access controller state and methods easily within your React components.
+
+## Usage
+
+1. Create an engine configuration
+
+Define the controllers you need in your engine configuration. This example includes `Summary` and `ProductList` controllers:
+
+```
+const config : CommerceEngineDefinitionOptions {
```

Comment:
In general, I'd add the imports in the samples, e.g. in this case:

```typescript
import {
  CommerceEngineDefinitionOptions,
  defineSummary,
  defineProductList,
  getSampleCommerceEngineConfiguration,
} from '@coveo/headless-react/ssr-commerce';

const config: CommerceEngineDefinitionOptions {
  ...
}
```

In this specific case it doesn't really change much, but what I'd like to help avoid is people accidentally importing stuff from `coveo/headless/ssr-commerce` instead of from `coveo/headless-react/ssr-commerce`. In particular, things like `defineCommercEngine`.